assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -142,10 +146,39 @@ const PageEdit = ( {
 		}
 	}, [ mediaType, mediaUrl ] );
 
+	// If there is more than one immovable block, leave only the last and remove the others.
+	useEffect( () => {
+		if ( immovableBlocks.length > 1 ) {
+			immovableBlocks.pop();
+			const blocksToRemove = immovableBlocks.map( ( { clientId: blockId } ) => blockId );
```

Comment:
There's no need to rename the variable here at all:

```suggestion
			const blocksToRemove = immovableBlocks.slice( 0, -1 ).map( ( { clientId } ) => clientId );
```